recipe/meta.yaml

```diff
@@ -12,15 +12,16 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<37]
```

Comment:
Why do you skip for Python < 3.7?

Reply:
The source repo requires Python >= 3.7.
https://github.com/pdm-project/pdm-pep517/blob/4bb714d52ff37cd80f44eee47bad42ef742e8332/pyproject.toml#L10